server/odc-service/src/main/java/com/oceanbase/odc/service/connection/util/ConnectionInfoUtil.java

```diff
@@ -71,6 +79,21 @@ public static String queryConnectionId(@NonNull Statement statement, @NonNull Di
         return ConnectionPluginUtil.getSessionExtension(dialectType).getConnectionId(statement.getConnection());
     }
 
+    public static String queryProxySessId(@NonNull Statement statement, @NonNull DialectType dialectType,
```

Comment:
queryOBProxySessId, this method name may be better?

Reply:
done